bzl_library.bzl

```diff
@@ -46,19 +46,19 @@ bzl_library = rule(
     implementation = _bzl_library_impl,
     attrs = {
         "srcs": attr.label_list(
-            allow_files = [".bzl"],
-            doc = "List of `.bzl` files that are processed to create this target.",
+            allow_files = [".bzl", ".scl"],
+            doc = "List of `.bzl` or `.scl` files that are processed to create this target.",
```

Comment:
Nit: Here and in rule doc: Maybe "`.bzl` and `.scl`" -- "and" instead of "or" to make clear it's not necessarily a homogeneous list of only .bzl or only .scl.

Reply:
Good point.